charts/authz/templates/deployment.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.name }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ .Values.labels.app }}
+    bank: {{ .Values.labels.bank }}
+    env: {{ .Values.labels.env }}
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: {{ .Values.labels.app }}
+      bank: {{ .Values.labels.bank }}
+      env: {{ .Values.labels.env }}
+  strategy:
```

Comment:
here we need to have to have a way to set default strategy
```
    type: RollingUpdate
    rollingUpdate:
      maxSurge: 2
```
and to be able to set it per env